tests/unit/StaffNotation.spec.ts

```diff
@@ -455,7 +455,7 @@ describe('StaffNotation', () => {
     sinon.restore()
   })
 
-  it('goes to limited restoration filing', async () => {
+  it('goes to limited restoration extension filing', async () => {
```

Comment:
According to line 469, this is testing a Limited Restoration filing.

You should have 4 tests in the Todo List test suite for verifying that all 4 restoration filing can be resumed.

Reply:
You were right. I was looking at the state filing.

Indeed, on line 493, you click the "extend limited restoration" button :)

Reply:
So this test suite LGTM, but please confirm/add the 4 Todo List tests.

Reply:
Yes. ToDoList tests in progress. Pushed changes by habit. Should have just stopped at commit :)

Reply:
Not at all. I think it's an excellent idea to "back up" your commits to GH by pushing them.

Unit tests LGTM.